lib/rails/html/sanitizer.rb

```diff
@@ -1,10 +1,51 @@
 # frozen_string_literal: true
 
 module Rails
-  module Html
-    class Sanitizer # :nodoc:
+  module HTML
+    class Sanitizer
+      class << self
+        def full_sanitizer
+          Rails::HTML4::FullSanitizer
+        end
+
+        def link_sanitizer
+          Rails::HTML4::LinkSanitizer
+        end
+
+        def safe_list_sanitizer
+          Rails::HTML4::SafeListSanitizer
+        end
+
+        def white_list_sanitizer # :nodoc:
+          safe_list_sanitizer
+        end
+
+        def html5_support?
+          unless @html5_support_set
+            @html5_support = Loofah.respond_to?(:html5_support?) && Loofah.html5_support?
+            @html5_support_set = true
+          end
+          @html5_support
+        end
+      end
+
       def sanitize(html, options = {})
-        raise NotImplementedError, "subclasses must implement sanitize method."
```

Comment:
This is a change in the API isn't? Before a subclass would only need to implement a method. Now it would need to reimplement 3. I think it is fine since subclasses that already implemented `sanitize` nothing would break.

Reply:
Ah, that's a fair point. I can move this default implementation into a concern as well -- so the base class would remain the same, but the first mixin would implement `sanitize` as a composition of parse/scrub/serialize. WDYT?

Reply:
I've updated the PR as I suggested above -- the base `Sanitizer` class is now unchanged.